### PR TITLE
Added Roadmap for Asio v0.1

### PR DESCRIPTION
**Pull request by yograterol**

### CODE REVIEW

**Comment by yograterol** [comment]
ping @echevemaster

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+MF - Major feature
+mf - Minor feature
+
+- [ ] Status Pages - v0.1
+  - [ ] Connect to NoSQL database (preferably MongoDB) - MF
+  - [ ] Create models for single domains. - MF
+  - [ ] Integrate an asynchronous task queue. - MF
+  - [ ] Ping to single domains (ports 80-443). (As asynchronous task called `ping_domain`) - MF
+  - [ ] Calculate response time. (Within `ping_domain`) - MF
+  - [ ] Get the HTML content and save URLs. - mf
```
Comment (echevemaster) [inline README.md]: this is named crawler

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+  - [ ] Find library for crawler. - mf
```
Comment (echevemaster) [inline README.md]: https://github.com/PuerkitoBio/gocrawl

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+  - [ ] Create events model, for records of ping. - MF
+  - [ ] Create REST interface (Define framework* - Options: Gorilla, Martini, Gin, Revel) - MF
+  - [ ] Create endpoints for manage and get info of single domain. - MF
```
Comment (echevemaster) [inline README.md]: just single domain? why not multiple of them?

File: README.md [hunk 4]
```diff
@@ -1 +1,33 @@
...
+  - [ ] Create endpoints for magage events (events are the ping result to a single domain). - mf
```
Comment (echevemaster) [inline README.md]: typo: manage not magage

File: README.md (hunk 3 above)
Comment (yograterol) [inline README.md]: For complex, is better start with one domain and after with different routes from a domain.

**Comment by yograterol** [comment]
Close for problem with new commits.

**Comment by echevemaster** [comment]
ok, we can merge?

**Comment by yograterol** [comment]
@echevemaster merge https://github.com/Clowl/asio/pull/4

**Pull request closed without merging**